src/SFML/Window/DRM/InputImplUDev.cpp

```diff
@@ -692,8 +692,8 @@ void InputImpl::setTerminalConfig()
 
     tcgetattr(STDIN_FILENO, &newTerminalConfig);          // get current terminal config
     oldTerminalConfig = newTerminalConfig;                // create a backup
-    newTerminalConfig.c_lflag &= ~(tcflag_t)ECHO;         // disable console feedback
-    newTerminalConfig.c_lflag &= ~(tcflag_t)ISIG;         // disable signals
+    newTerminalConfig.c_lflag &= ~static_cast<tcflag_t>(ECHO);         // disable console feedback
```

Comment:
Should this be rewritten as such?
```cpp
newTerminalConfig.c_lflag &= static_cast<tcflag_t>(~ECHO);
```

Reply:
I do't believe so because the cast should be occurring before the `~` is applied.

Reply:
Alrighty